app/models/notification.rb

```diff
@@ -64,10 +68,10 @@ def self.purge_old!
   end
 
   def self.ensure_consistency!
-    DB.exec(<<~SQL, Notification.types[:private_message])
+    DB.exec(<<~SQL)
       DELETE
         FROM notifications n
-       WHERE notification_type = ?
+       WHERE high_priority = TRUE
```

Comment:
`where high_priority` should do the trick, I think the capital true is a bit too verbose.